tests/expr_and_series/nth_test.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from typing import Any
+
+import polars as pl
+import pytest
+
+import narwhals.stable.v1 as nw
+from narwhals.utils import parse_version
+from tests.utils import Constructor
+from tests.utils import compare_dicts
+
+data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.1, 8, 9]}
+
+
+@pytest.mark.parametrize(
+    ("idx", "expected"),
+    [
+        (0, {"a": [1, 3, 2]}),
+        ([0, 1], {"a": [1, 3, 2], "b": [4, 4, 6]}),
+        ([0, 2], {"a": [1, 3, 2], "z": [7.1, 8, 9]}),
+    ],
+)
+def test_nth(
+    constructor: Constructor,
+    idx: int | list[int],
+    expected: dict[str, list[int]],
+    request: Any,
+) -> None:
+    if "polars" in str(constructor) and parse_version(pl.__version__) < parse_version(
+        "0.20.26"
+    ):
+        request.applymarker(pytest.mark.xfail)
+    df = nw.from_native(constructor(data))
+    result = df.select(nw.nth(idx))
+    compare_dicts(result, expected)
+
+
+@pytest.mark.skipif(
+    parse_version(pl.__version__) >= parse_version("0.20.26"),
+    reason="only riase error for Polars versions < 0.20.26",
+)
+def test_nth_not_supported() -> None:
```

Comment:
This appear to not be covered as it does not always run, and coveragepy does not pick it up

```suggestion
def test_nth_not_supported() -> None:  # pragma: no cover
```